downstream/modules/platform/ref-using-custom-tls-certificates.adoc

```diff
@@ -0,0 +1,48 @@
+:_newdoc-version: 2.15.1
+:_template-generated: 2024-01-12
+
+:_mod-docs-content-type: REFERENCE
+
+[id="using-custom-tls-certificates_{context}"]
+= Using custom TLS certificates
+
+[role="_abstract"]
+
+By default, the installer generates TLS certificates and keys for all services which are signed by a custom Certificate Authority (CA). You can provide a custom TLS certificate/key for each service. If that certificate is signed by a custom CA, you will need to provide the CA TLS certificate and key.
```

Comment:
Suggest changing "will need to" to "must". IBM Style: Use "must", not "have to" or "need to"